src/data/builds/hunter/5200_hun_trapping_100.ts

```diff
@@ -0,0 +1,44 @@
+import { Class } from "~/composables/Characters";
+
+import { Build } from "~/composables/Builds";
+
+export const build: Build = {
+  title: "Hunter Trapping (~100)",
+  notes: "",
+  version: "1.23",
+  level: 50,
```

Comment:
Is the level here correct? The title says ~100, but the leve field says 50. The numeric field might be used in the future to sort/filter builds by level, so I want to make sure these are accurate as we update them.

Reply:
Yep, that's meant to be 100, not 50.